library/src/math/align.rs

```diff
@@ -28,21 +28,31 @@ pub(super) fn alignments(rows: &[MathRow]) -> AlignmentResult {
     for row in rows {
         let mut width = Abs::zero();
         let mut alignment_index = 0;
-        for fragment in row.iter() {
-            if matches!(fragment, MathFragment::Align) {
+
+        macro_rules! update_width {
```

Comment:
could you try to write this without a macro?

Reply:
Leads to code duplication which is a bummer, but sounds good.